gen/aircraft.py

```diff
@@ -533,6 +535,17 @@ class AircraftData:
     #: Defines how channels should be named when printed in the kneeboard.
     channel_namer: Type[ChannelNamer] = ChannelNamer
 
+@dataclass
+class AewcInfo:
+    """AEW&C information for the kneeboard."""
+    freq: RadioFrequency
+    depature_location: str
+    depature_time: Optional[timedelta]
+    arrival_time: Optional[timedelta]
+
+@dataclass
+class CustomAirSupport:
+    aewc: List[AewcInfo] = field(default_factory=list)
```

Comment:
And with the previous comment done you don't need this, you can just add it to the existing `awacs` list in `AirSupport`.